packages/contracts/test/contracts/L1/deployment/ChugSplashDictator.spec..ts

```diff
@@ -1,51 +1,40 @@
-/* External Imports */
```

Comment:
What's up with the `..ts` file extension?

Reply:
Lol, surprised we never noticed that.